quickstart/101-analysis-services-create/variables.tf

```diff
@@ -0,0 +1,24 @@
+variable "resource_group_location" {
+  default     = "eastus"
```

Comment:
These variables should define the type for the value. I think most of them are string but without the type defined, a user can pass in any type which is not correct.